src/Console/Application.php

```diff
@@ -131,17 +132,26 @@ protected function doRunCommand(Command $command, InputInterface $input, OutputI
 	 * Check for owncloud instance
 	 * @throws \RuntimeException
 	 */
-	protected function assertOwncloudFound(){
+	protected function assertOwnCloudFound(){
 		$container = $this->getContainer();
+		/** @var Locator $locator */
 		$locator = $container['utils.locator'];
-		$pathToVersionFile = $locator->getPathToVersionFile();
-		if (!file_exists($pathToVersionFile) || !is_file($pathToVersionFile)){
-			throw new \RuntimeException('ownCloud is not found in ' . dirname($pathToVersionFile));
+
+		$file = $locator->getPathToVersionFile();
+		if (!file_exists($file) || !is_file($file)){
+			throw new \RuntimeException('ownCloud is not found in ' . dirname($file));
+		}
+
+		// assert minimum version
+		$installedVersion = implode('.', $locator->getInstalledVersion());
+		if (version_compare($installedVersion, '9.0.0', '<')) {
+			throw new \RuntimeException("Minimum ownCloud version 9.0.0 is required for the updater - $installedVersion was found in " . $locator->getOwncloudRootPath());
 		}
 
-		$pathToOccFile = $locator->getPathToOccFile();
-		if (!file_exists($pathToOccFile) || !is_file($pathToOccFile)){
-			throw new \RuntimeException('ownCloud is not found in ' . dirname($pathToOccFile));
+		// has to be installed
+		$file = $locator->getPathtoConfigFile();
+		if (!file_exists($file) || !is_file($file)){
+			throw new \RuntimeException('ownCloud in ' . dirname($file) . ' is not installed.');
```

Comment:
@Deepdiver1975 dirname(dirname($file)) unless you want something that ends with `/config/` here?